src/univariate/discrete/betabinomial.jl

```diff
@@ -85,17 +85,17 @@ function pdf(d::BetaBinomial, k::Int)
     chooseinv = (n+1) * beta(k+1, n-k+1)
     numerator = beta(k + α, n - k + β)
     denominator = beta(α, β)
-    return (numerator / denominator) / chooseinv
+    return numerator / (denominator * chooseinv)
 end
 
 function pdf(d::BetaBinomial)
     n, α, β = d.n, d.α, d.β
     denominator = beta(α, β)
-    values = Array{Float64, 1}()
-    for k = 0:n
+    values = Array(Float64, n+1)
```

Comment:
The `N` part is not necessary though. `Array{T}(n+1)` is fine since `n+1` implicitly determines `N`.